SecretSanta/SecretSanta.Business.Gift.Tests/BusinessTests.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework.Internal;
+using SecretSanta.Business;
+using System.Collections.Generic;
+
+namespace SecretSanta.Business.Tests
+{
+    [TestClass]
+    public class BusinessTests
+    {
+        [TestMethod]
+        //Test methods can contain underscores
+#pragma warning disable CA1707 // Identifiers should not contain underscores
+        public void Create_Gift_Success()
+#pragma warning restore CA1707 // Identifiers should not contain underscores
+        {
+            //arrange
+            int id = 0;
+            string title = "title";
+            string desc = "description";
+            string url = "URL";
+            User user = new User(1, "fname", "lname");
+            //act
+            Gift gift = new Gift(id, title, desc, url, user);
+            //assert
+            Assert.AreEqual<int>(id, gift.Id , "id value is unexpected");
+            Assert.AreEqual<string>(title, gift.Title, "title value is unexpected");
+            Assert.AreEqual<string>(desc, gift.Description, "description value is unexpected");
+            Assert.AreEqual<string>(url, gift.Url, "url value is unexpected");
+            Assert.AreEqual<User>(user, gift.User, "User value is unexpected");
+        }
+        [TestMethod]
+#pragma warning disable CA1707 // Identifiers should not contain underscores ###Test method fine
+        public void Create_User_Success()
+#pragma warning restore CA1707 // Identifiers should not contain underscores
+        {
+            //arrange
+            int id = 0;
+            string fname = "fname";
```

Comment:
```suggestion
            string firstName = "fname";
```
I personally would suggest avoiding abbreviations. The name may cause confusion for someone looking at this code with no prior knowledge of the project. (Eg. No knowledge that User contains a first and last name property making fname a confusing variable name).